xml/issue3043.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version='1.0' encoding='utf-8' standalone='no'?>
+<!DOCTYPE issue SYSTEM "lwg-issue.dtd">
+
+<issue num="3043" status="New">
+<title>Bogus postcondition for <tt>filesystem_error</tt> constructor</title>
+<section><sref ref="[fs.filesystem_error.members]"/></section>
+<submitter>Tim Song</submitter>
+<date>6 Dec 2017</date>
+<priority>99</priority>
+
+<discussion>
+<p>
+<sref ref="[fs.filesystem_error.members]"/> says that constructors of
+<tt>filesystem_error</tt> has the postcondition <tt>runtime_error::what()</tt> has the
```

Comment:
"constructors...has" should be "have"

This sentence would also be easier to parse with quotes around the quoted text.

Reply:
It's a table, not exactly a quote. Will fix grammar in a sec.

Reply:
Clarifying:

> ...constructors of <tt>filesystem_error</tt> have the postcondition <tt>runtime_error::what()</tt> has the value <tt>what_arg.c_str()</tt>.

reads as "foo have the bar has the baz." It needs quotes:

> ...constructors of <tt>filesystem_error</tt> have the postcondition "<tt>runtime_error::what()</tt> has the value <tt>what_arg.c_str()</tt>".

Reply:
I added an extra "that" after "postcondition" instead.